src/cuga/backend/cuga_graph/nodes/cuga_lite/nl_auto_continue_classifier.py

```
@@ -29,6 +29,59 @@
+    r"(?:need\s+to|have\s+to|should|must|will|'ll|going\s+to|gonna|"
+    r"start\s+by|begin\s+by|"
+    r"search|discover|find|look\s+up|fetch|call|query|inspect|"
+    r"explore|examine|check|investigate|figure\s+out|determine|"
+    r"retrieve|gather|list|enumerate)\b",
+    re.IGNORECASE,
+)
+
+# A negation usually marks a result or refusal ("I could not find …"), not a
+# forward-looking plan — let those fall through to the LLM classifier / finalize.
+_NEGATION_RE = re.compile(
+    r"\b(?:not|never|unable|cannot|no)\b|\w+n['’]t\b",
```

Ruff (0.15.20)

[warning] String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)